apps/web/src/components/Composer/Post/New.tsx

```diff
@@ -19,11 +19,19 @@ const NewPost: FC = () => {
   const setPublicationContent = usePublicationStore(
     (state) => state.setPublicationContent
   );
+  const setShowNewSpacesModal = useGlobalModalStateStore(
+    (state) => state.setShowNewSpacesModal
+  );
 
   const openModal = () => {
     setShowNewPostModal(true);
```

Comment:
It would be good to have type of modal here itself instead of calling different states, something like

```ts
setShowNewPostModal(true, PublicationType.Spaces);
```

Reply:
Updated